scipy/stats/tests/test_marray.py

```diff
@@ -0,0 +1,126 @@
+import pytest
+import numpy as np
+from scipy import stats
+
+from scipy._lib._array_api import xp_assert_close, xp_assert_equal
+from scipy.stats._stats_py import _xp_mean, _xp_var
+skip_backend = pytest.mark.skip_xp_backends
+
+
+def get_arrays(n_arrays, *, dtype='float64', xp=np, shape=(7, 8), seed=84912165484321):
+    marray = pytest.importorskip('marray')
```

Comment:
Can this be a module-level skip rather than within any test? Faster and easier to understand for readers new to this file.

Reply:
Definitely. Somehow I just learned about`importorskip` and didn't see that existed.

Reply:
If we don't end up changing anything else, I'll do this in the follow-up that adds hypothesis tests.